### PR TITLE
Add section documenting the deprecated WCF integration

### PR DESCRIPTION
**Pull request by timbussmann**
relates to https://github.com/Particular/NServiceBus.Host/pull/103

Further plans as stated [here](https://github.com/Particular/V6Launch/issues/122#issuecomment-245542817) are to make a wcf integration package available under the particular labs repository. At that point we can point to this repository on this section. /cc @danielmarbach

### CODE REVIEW

File: nservicebus/upgrades/host-6to7.md [hunk 1]
```diff
@@ -40,3 +40,8 @@ The `IMessageSession` parameter provides all the necessary methods to send messa
 include:5to6removePShelpers
 
 WARNING: If an `EndpointConfig.cs` file already exists in the project, be careful to not overwrite it when upgrading the `NServiceBus.Host` package. If Visual Studio detects a conflict, it will ask whether the file should be overwritten. To keep the old configuration, choose `No`.
+
+
+## WCF Integration
+
+The WCF integration using `WcfService` has been removed from the host. The [WCF request response via Callbacks](http://docs.particular.net/samples/web/wcf-callbacks/) sample shows how to expose message handlers over WCF services using NServiceBus Version 6 and the NServiceBus.Callbacks package.
```
Comment (SimonCropp) [inline nservicebus/upgrades/host-6to7.md]: Links should not be qualified with the domain

Comment (timbussmann) [inline nservicebus/upgrades/host-6to7.md]: yeah the build already shouted at me :+1:

**Comment by indualagarsamy** [comment]
@timbussmann - does this guidance change based on the proposed plan of action here:
https://github.com/Particular/V6Launch/issues/122#issuecomment-245542817

If so can you close this PR?

**Comment by indualagarsamy** [comment]
@timbussmann - Also this document also needs to be updated:
http://docs.particular.net/nservicebus/hosting/expose-an-endpoint-as-a-web-wcf-service
We have an related issue for that. See: https://github.com/Particular/docs.particular.net/issues/1472

File: nservicebus/upgrades/host-6to7.md [hunk 2]
```diff
@@ -40,3 +40,8 @@ The `IMessageSession` parameter provides all the necessary methods to send messa
 include:5to6removePShelpers
 
 WARNING: If an `EndpointConfig.cs` file already exists in the project, be careful to not overwrite it when upgrading the `NServiceBus.Host` package. If Visual Studio detects a conflict, it will ask whether the file should be overwritten. To keep the old configuration, choose `No`.
+
+
+## WCF Integration
+
+The WCF integration using `WcfService` has been removed from the host. The [WCF request response via Callbacks](/samples/web/wcf-callbacks) sample shows how to expose message handlers over WCF services using NServiceBus Version 6 and the NServiceBus.Callbacks package.
```
Comment (SimonCropp) [inline nservicebus/upgrades/host-6to7.md]: as previously highlighted by @DavidBoike this sample is not a adequate replacement for the WCF code int he host, and it was never intended to be. The reason is that it does not allow client to generate the proxy off the wcf serivice.

Comment (timbussmann) [inline nservicebus/upgrades/host-6to7.md]: @SimonCropp as stated in the PR description, this is currently (until the external package is available!) the only option available and other WCF integration is just deprecated. I'm happy to reword the text here to state that more clearly and again, I expect this text to change as soon as we can provide the external package/library for doing "proper" WCF integration as soon as it is available.

**Comment by timbussmann** [comment]
> does this guidance change based on the proposed plan of action here:
Particular/V6Launch#122 (comment)

that's why I raised this PR

> Also this document also needs to be updated: [...] We have an related issue for that. See: #1472

so this can be handled by the mentioned issue I assume?

**Comment by SimonCropp** [comment]
I would prefer 1472 was handled in this PR

**Comment by timbussmann** [comment]
> I would prefer 1472 was handled in this PR

@SimonCropp how would you handle it? Add a notice about this being deprecated in v7 of the host?

**Comment by indualagarsamy** [comment]
@timbussmann - I would expect for starters that the v6 code snippets be removed, because its misleading. And add a notice about it being deprecated at the top, which would include the link to the guidance on what the user needs to do.

**Comment by timbussmann** [comment]
>  I would expect for starters that the v6 code snippets be removed

why? it's v6 of the Host. The Core v6 equivalent is NSB.Host v7, which is not included there.

> And add a notice about it being deprecated at the top

:+1:

File: nservicebus/hosting/expose-an-endpoint-as-a-web-wcf-service.md [hunk 3]
```diff
@@ -10,6 +10,8 @@ related:
  - samples/web/wcf-callbacks
 ---
 
+Note: NServiceBus Version 6 and above no longer provide built-in support for WCF integration. Refer to the [Upgrade Guide](/nservicebus/upgrades/host-6to7) for further information.
```
Comment (indualagarsamy) [inline nservicebus/hosting/expose-an-endpoint-as-a-web-wcf-service.md]: I think this should be warning instead. And a minor suggestion:

WARNING: Starting from NServiceBus Core Versions 6 and above, the built-in support for WCF integration has been removed.  For more details on how to use WCF integration for these versions, refer to the [Upgrade Guide](/nservicebus/upgrades/host-6to7).

Comment (timbussmann) [inline nservicebus/hosting/expose-an-endpoint-as-a-web-wcf-service.md]: I'm pretty sure we used NOTE for deprecation notice in other places of the docs. :+1: on the suggested text though

**Pull request merged**